pkg/sns/SnsConnectionFactory.php

```diff
@@ -71,7 +72,7 @@ public function __construct($config = 'sns:')
      */
     public function createContext(): Context
     {
-        return new SnsContext($this->establishConnection(), $this->config);
+        return new SnsContext($this->establishConnection(), $this->config, $this->config['topic_arns']);
```

Comment:
why do you pass topc_arns as separate arg? They are already there in config which is passed as second arg

Reply:
removed third arg